WordPress/src/jetpack/assets/reader_text_events.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * wvHandler is the name of the message handler in the webview, added via createJsObject inside
+ * ReaderPostRenderer. It handler the `postMessage` calls from the WebView.
```

Comment:
💡 Nitpick: I think there's a typo. Should it be "It handles" instead of "It handler"?